packages/modules/manage-versions/tsconfig.json

```diff
@@ -1,14 +1,12 @@
 {
   "compilerOptions": {
-    "target": "es2017",
+    "target": "ES2020",
     "lib": [
-      "esnext.asynciterable",
-      "es2017",
-      "dom",
-      "dom.iterable",
-      "scripthost"
+        "dom",
+        "dom.iterable",
+        "esnext"
```

Comment:
do we use any of that new features?

Reply:
By default I believe tsconfig has these in lib